src/CommonDBTM.php

```diff
@@ -6590,11 +6590,6 @@ public static function displayFullPageForItem(
         $item->display($options);
         Profiler::getInstance()->stop(static::class . '::display');
 
-        // Display extra html if needed
-        if (!empty($options['after_display'] ?? "")) {
-            echo $options['after_display'];
-        }
-
```

Comment:
This was used only by `front/problem.form.php` and `front/ticket.form.php`, and was now a security flaw since the request is not anymore autosanitized.